gammapy/time/tests/test_lightcurve.py

```diff
@@ -28,3 +35,61 @@ def test_lightcurve_fvar():
 def test_lightcurve_plot():
     lc = LightCurve.simulate_example()
     lc.plot()
+
+
+@requires_data('gammapy-extra')
+@pytest.fixture(scope='session')
+def spec_extraction():
+    data_store = DataStore.from_dir('$GAMMAPY_EXTRA/datasets/hess-crab4-hd-hap-prod2/')
+    obs_ids = [23523, 23526, 23559, 23592]
```

Comment:
How long does this test run?
Maybe restrict to two or three runs if it takes long?